azerothcore/azerothcore-wotlk
# fix(DB): Use sniffed values for Midsummer Music Doodad spawns

unfortunately one spawn is missing (guid 50564)

## Changes Proposed:

This PR proposes changes to:

This PR includes the proposed changes from the issue plus additional entries in game_event_gameobject.

To me it seems that the missing entries in game_event_gameobject led to Music not always beeing played as the GameObjectAI script checks if Midsummer event is active.
So without game_event_gameobject entries the GameObject immediately spawns - Midsummer event not active - GameObjectAI not playing music even if Midsummer Event gets started afterwards.

Therefore the proposed changes in the issue were correct but not complete.

## Issues Addressed:

- Closes #12250 

## SOURCE:

The changes have been validated through:

## Tests Performed:

This PR has been:


## How to Test the Changes:



.go xyz 9794.82 -7248.74 26.0974 530
.go xyz -3798.69 -11509 -134.821 530
.go xyz -8836.6 866.36 98.7168 0

before PR:
observe no Midsummer music playing

after PR:
observe Midsummer Music playing

## Known Issues and TODO List:


- [ ] unfortunately one spawn is missing (guid 50564)


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
